src/tests/t_tokenize.c

```diff
@@ -18,10 +18,16 @@ static const char *test_tokens[] = {
 	"some-term-1", "another-term-2", "another-term-2"
 };
 
-static const char test_text[] = "The quick brown fox jumped over the lazy dog";
+static const char test_text[] = "The quick brown fox jumped over the lazy dog. year-end for I.B.M.";
```

Comment:
You can just an array of test strings (with an array of expected string below):
```c
static const char *test_text[] = {
	"first string",
	"second string",
	...
};
```
Then just iterate them in the test. Note: for the tests, it's perfectly fine to switch to strlen() as tests are not performance sensitive: whatever makes them more readable.